src/dashboard/src/components/access/views.py

```diff
@@ -81,15 +81,31 @@ def all_records(client, request, system=''):
 
 
 @_authenticate_to_archivesspace
-def get_record(client, request, system='', record_id=''):
-    record_id = record_id.replace('-', '/')
+def record(client, request, system='', record_id=''):
+    if request.method == 'PUT':
+        try:
+            new_record = json.load(request)
+        except ValueError:
+            response = {
+                'success': False,
+                'message': 'No JSON object could be decoded from request body.',
+            }
+            return helpers.json_response(response)
 
-    try:
-        records = client.get_resource_component_and_children(record_id,
-                                                             recurse_max_level=3)
-        return helpers.json_response(records)
-    except ArchivesSpaceError as e:
-        return helpers.json_response({"error": True, "message": str(e)})
+        client.edit_record(new_record)
+        return helpers.json_response({
+            'success': True,
+            'message': 'Record updated.',
+        })
+    elif request.method == 'GET':
+        record_id = record_id.replace('-', '/')
+
+        try:
+            records = client.get_resource_component_and_children(record_id,
+                                                                 recurse_max_level=3)
+            return helpers.json_response(records)
+        except ArchivesSpaceError as e:
+            return helpers.json_response({"error": True, "message": str(e)})
```

Comment:
Should this return a non-200 status code?  Also, here the response has `"error": True`, while above it has `'success': False` - make the API consistent?